ovos_adapt/opm.py

```
@@ -47,12 +48,13 @@ class AdaptPipeline(PipelinePlugin):
     def __init__(self, config=None):
         core_config = Configuration()
-        self.config = config or core_config.get("context", {})  # legacy mycroft-core path
-        self.lang = core_config.get("lang", "en-us")
+        config = config or core_config.get("context", {})  # legacy mycroft-core path
+        super().__init__(config)
+        self.lang = standardize_lang_tag(core_config.get("lang", "en-US"))
         langs = core_config.get('secondary_langs') or []
         if self.lang not in langs:
             langs.append(self.lang)
-
+        langs = [standardize_lang_tag(l) for l in langs]
```

Ruff

Ambiguous variable name: `l`

(E741)